### ISSUE TITLE
Syntax Rule Issue: Class properties defined with constructor argument are reported to be missing

### ISSUE THREAD

**Issue by atsu85** [MeirionHughes/aurelia-template-lint#61]
> ERROR: cannot find 'constructorPublicField' in type 'ConstructorFieldCustomElement'

```html
${constructorPublicField}
```
```TypeScript
export class ConstructorFieldCustomElement {
    constructor(public constructorPublicField:string){}
}
```

**Comment by atsu85** [comment]
I'll send a PR and test soon
